src/Microsoft.VisualStudio.ProjectSystem.Managed.VS/ProjectSystem/VS/Rename/Renamer.cs

```diff
@@ -91,7 +91,7 @@ internal async Task HandleRenameAsync(string oldFilePath, string newFilePath)
                 return;
 
             // Ask if the user wants to rename the symbol
-            if (await CheckUserConfirmation(oldName))
+            if (await CheckUserConfirmation(oldName)) != true)
```

Comment:
I probably broke this due to misreading this check, IMO this reads a little better:

``` C#
if (!await CheckUserConfirmation(oldName))
```

Reply:
Or maybe

``` C#
bool userWantsToRenameSymbol = await CheckUserConfirmation(oldName);
if (!userWantsToRenameSymbol)
    ...
```

`!await` does look a little funny to me...